calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
I went with meta data because that is what the docker run command help calls that data, I did have a note in one of my revisions like `(as set with --label)` but removed it.
I'm good with going back to calling them labels but just wanted to put some reasoning behind what I did.
The pulling selectors was referring to them being pulled from the Docker container metadata, I like your phrasing of them being 'Extracted'.